wiki/History_of_osu!/Online_rankings/osu!catch/en.md

```diff
@@ -0,0 +1,100 @@
+---
+tags:
+  - history
+  - leaderboard
+  - rank 1
+  - "rank #1"
+  - rankings
+  - top player
+---
+
+# osu!catch rankings history
+
+The following is a list of players who are recognised to have reached rank #1 in the [global ranking](https://osu.ppy.sh/rankings/fruits/performance) of [osu!catch](/wiki/Game_mode/osu!catch).
+
+Over the years, there were three different systems used for ranking players:
+
+- [Ranked Score](/wiki/Gameplay/Score/Ranked_score) (6 October 2007 – 24 July 2012)
+- [ppv1](/wiki/Performance_points/ppv1) (24 July 2012 – 27 January 2014)
+- [ppv2](/wiki/Performance_points) (27 January 2014 – present)
+
+Because of this, this list only includes players who have reached rank #1 on the main leaderboard that was used at the time. *Italicised* dates and numbers indicate that the corresponding player is continuing their reign as the osu!'s rank #1 holder either from the previous year or into the following year.
+
+## 2018
+
+| Player | From | To | Sources |
+| --: | :-- | :-- | :-- |
+| ::{ flag=RU }:: [skvix](https://osu.ppy.sh/users/4447639) | *2018-04-14* | *2019-04-06* | [\[1\]](https://web.archive.org/web/20180414205044/https://osu.ppy.sh/rankings/fruits/performance) |
+
+## 2019
+
+| Player | From | To | Sources |
+| --: | :-- | :-- | :-- |
+| ::{ flag=RU }:: [skvix](https://osu.ppy.sh/users/4447639) | *2018-04-14* | 2019-04-06 | [\[1\]](https://web.archive.org/web/20180414205044/https://osu.ppy.sh/rankings/fruits/performance) |
+| ::{ flag=KR }:: [Motion](https://osu.ppy.sh/users/3885626) | 2019-04-06 | 2019-08-09 | [\[1\]](https://web.archive.org/web/20190406161641/https://osu.ppy.sh/rankings/fruits/performance) |
+| ::{ flag=KR }:: [YesMyDarknesss](https://osu.ppy.sh/users/4158549) | 2019-08-09 | 2019-08-18 | [\[1\]](https://www.reddit.com/r/osugame/comments/co0ruo/after_2_years_224hyperw7_has_reclaimed_rank_1_on/) [\[2\]](https://web.archive.org/web/20190810115254/https://osu.ppy.sh/rankings/fruits/performance) |
```

Comment:
this should either use their name at the time (`[224]Hyperw7`) or a combination of current and former name (`YesMyDarknesss (*[224]Hyperw7*)`). the latter is what the osu page does so probs do that

also i did not check all of the users here for this so there may be more.